indico/modules/events/abstracts/models/judgements.py

```diff
@@ -0,0 +1,105 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from sqlalchemy.ext.declarative import declared_attr
+
+from indico.core.db.sqlalchemy import db, UTCDateTime
+from indico.core.db.sqlalchemy.util.models import auto_table_args
+from indico.util.string import format_repr, return_ascii
+
+
+class Judgement(db.Model):
+    """Represents an abstract that can be associated to a Contribution."""
+
+    __tablename__ = 'judgements'
+    __auto_table_args = (db.Index(None, 'abstract_id', 'track_id', 'user_id', unique=True),
+                         {'schema': 'event_abstracts'})
+
+    @declared_attr
+    def __table_args__(cls):
+        return auto_table_args(cls)
+
+    id = db.Column(
+        db.Integer,
+        primary_key=True
+    )
+    creation_dt = db.Column(
+        UTCDateTime,
+        nullable=False,
+        index=True
+    )
+    abstract_id = db.Column(
+        db.Integer,
+        db.ForeignKey('event_abstracts.abstracts.id'),
+        index=True,
+        nullable=False
+    )
+    track_id = db.Column(
+        db.Integer,
+        nullable=False
+    )
+    user_id = db.Column(
```

Comment:
Thought of it, but wanted to reinforce that this is a user_id like any other in the DB.